lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -787,6 +800,25 @@ spec = do
                     }
             in
                 x' === x .&&. show x' === show x
+        it "ApiNetworkParameters" $ property $ \x ->
+            let
+                x' = ApiNetworkParameters
+                    { genesisBlockHash =
+                            genesisBlockHash (x :: ApiNetworkParameters)
+                    , blockchainStartTime =
+                            blockchainStartTime (x :: ApiNetworkParameters)
+                    , slotLength =
+                            slotLength (x :: ApiNetworkParameters)
+                    , epochLength =
+                            epochLength (x :: ApiNetworkParameters)
+                    , epochStability =
+                            epochStability (x :: ApiNetworkParameters)
+                    , activeSlotCoefficient =
+                            activeSlotCoefficient (x :: ApiNetworkParameters)
+                    }
+            in
+                x' === x .&&. show x' === show x
```

Comment:
> "pointless tests to trigger coverage for record accessors"

Hm, ok 👍 

Unrelated to the PR, but do we still care about coverage to this extent? We have removed the badge from the readme at least.

Reply:
yeah, `pointless tests` title says it all